src/feature/controls_panel/ControlsTabView.tsx

```diff
@@ -86,7 +86,11 @@ const ControlsTab = ({ results }: ControlsTabProps) => {
     // }
   };
 
-  const updateCtrlValue = (val: string, ctrl: CtlManifestType) => {
+  const updateCtrlValue = (
+    val: string,
+    ctrl: CtlManifestType,
+    camelCase: ParamValueType
```

Comment:
I apologize that my previous comment was not clear to you, I meant to change `ValType` to match the format `camelCase` as follows: `valType`

Reply:
Sorry, my bad for not asking for a clarification, I didn't know what camelCase meant and just assumed it was a rename.

Reply:
No problem at all, `camelCase` is indeed a naming convention commonly used in programming, [This article](https://dev.to/prahladyeri/underscores-camelcasing-and-pascalcasing-the-three-naming-conventions-every-programmer-should-be-aware-of-3aed) could be helpful for you to understand naming conventions used in programming.